### DISCUSSION TITLE
Maxima 22.0.5_02 crashes on MacOs (tried both Monterey and Bigsur) when opening Preferences

### DISCUSSION THREAD

**Discussion by mlomonaco** [wxMaxima-developers/wxmaxima#1701] [Q&A]
Error given in the console after launching the command `wxmaxima` is `Abort trap: 6`

**Comment by gunterkoenigsmann** [comment]
For the next release I have resigned big parts of the dialogue.I hope whilst doing that this problem has resolved.

**Reply by mlomonaco** [reply]
Circling back on this after a while, on version 23.04 now it crashes same way but upon _closing_ the preferences dialog.
